tests/fuzz/fuzz_server.c

```diff
@@ -0,0 +1,178 @@
+#include "mbedtls/ssl.h"
+#include "mbedtls/entropy.h"
+#include "mbedtls/ctr_drbg.h"
+#include "mbedtls/certs.h"
+#include "mbedtls/ssl_ticket.h"
+#include "common.h"
+#include <string.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include <stdint.h>
+
+
```

Comment:
Please wrap all the code around `#if defined(MBEDTLS_SSL_SRV_C)`, similar to gating the code of `fuzz_x509_crt.c` on `MBEDTLS_X509_CRT_PARSE_C`. Otherwise the build fails when `MBEDTLS_SSL_SRV_C` is turned off in the configuration.

Likewise in the other TLS executables (`MBEDTLS_SSL_CLI_C` for TLS clients).

Reply:
ok done